db/db_versioned.go

```diff
@@ -80,50 +96,192 @@ func (b *BoltDBVersioned) Stop(ctx context.Context) error {
 	return b.db.Stop(ctx)
 }
 
+// Base returns the underlying KVStore
+func (b *BoltDBVersioned) Base() KVStore {
+	return b.db
+}
+
 // Put writes a <key, value> record
-func (b *BoltDBVersioned) Put(ns string, version uint64, key, value []byte) error {
+func (b *BoltDBVersioned) Put(version uint64, ns string, key, value []byte) error {
```

Comment:
Do we really need to implement these single function`Put/Delete`? I prefer only keep `WriteBatch`

Reply:
discussed offline, it does not hurt to keep it